package.json

```diff
@@ -3,11 +3,17 @@
   "version": "1.0.0",
   "description": "",
   "author": "",
+  "scripts": {
+    "dev": "node server.js --testing",
+    "prod": "sudo node server.js"
+  },
   "dependencies": {
     "es6-promise": "^3.2.1",
     "less": "^2.7.1",
-    "restify": "^4.0.4",
+    "letsencrypt-express": "^1.1.5",
```

Comment:
It sucks that you had to do all this. I had it in master, but not as of when you forked.